### ISSUE TITLE
[Feature Requests] Configuration Options and Support for plurals and context

### ISSUE THREAD

**Issue by frudolph77** [marhali/easy-i18n#93]
Hello
I'm trying out some i18n plugins for IntelliJ in combination with `i18next(-react)`, and I found yours and it's one of the better. 

In `i18next` the default namespace separator is `:`  or the `ns`-attribute in the parameter object.
e.g. `t('common:submit')`  resp. `t('submit', { ns: 'common' })`

Therefore I'm missing some, in my opinion, useful (configuration) features:
- The namespace, key, plurals and context separators should be customizable.
- an option to define a default namespace, which then is not given in the key e.g. `t('submit')` where common is now the default ns.

It would also be great when you add support for plurals and context

### Plurals, Context and both combined
```json
{
  "keyWithCount_one": "{{count}} item",
  "keyWithCount_other": "{{count}} items"

  "friend": "A friend",
  "friend_male": "A boyfriend",
  "friend_female": "A girlfriend"

  "friend_male_one": "A boyfriend",
  "friend_female_one": "A girlfriend",
  "friend_male_other": "{{count}} boyfriends",
  "friend_female_other": "{{count}} girlfriends"
}
```

```js
i18next.t('keyWithCount', {count: 0}); // -> "0 items"
i18next.t('keyWithCount', {count: 1}); // -> "1 item"
i18next.t('keyWithCount', {count: 5}); // -> "5 items"

i18next.t('friend'); // -> "A friend"
i18next.t('friend', {context: 'male' }); // -> "A boyfriend"
i18next.t('friend', {context: 'female' }); // -> "A girlfriend"

i18next.t('friend', {context: 'male', count: 1}); // -> "A boyfriend"
i18next.t('friend', {context: 'female', count: 1}); // -> "A girlfriend"
i18next.t('friend', {context: 'male', count: 100}); // -> "100 boyfriends"
i18next.t('friend', {context: 'female', count: 100}); // -> "100 girlfriends"
```
So if there is a `count`, `context` or both of them in the parameter object if would be great to see all matching keys in the hover.
Be aware of, that there are languages with at least 5 plurals besides the singular.  

Kind Regards
Frank

**Comment by marhali** [comment]
Thanks for your report 👍 

Individual adaption to framework specific usecases are not in the main scope right now. I will keep this as an enhancement for long-term features.

**Comment by marhali** [comment]
Public testing for the first release candidate is underway. 
Feel free to try it out and take a look. Feedback is always welcome 😃.
 
The release will be available in the RC channel soon. Alternatively, you can download it [manually](https://github.com/marhali/easy-i18n/releases/tag/v4.0.0-rc.1).

**Comment by marhali** [comment]
Resolved by release v4.0.0
